common/modifiers/wc_lifestyle_modifiers.txt

```diff
@@ -24,4 +24,10 @@ master_of_shadows_modifier = {
 	icon = intrigue_positive
 	prowess = 1
 	stacking = yes
+}
+
+stasis_trap_stunned = {
+	icon = intrigue_negative
+
+	disables_combat_leadership = yes
```

Comment:
It doesn't work.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120221302-39089200-c24f-11eb-980e-0e4e74e1cb26.png)

</details>